zowex/src/main.rs

```diff
@@ -301,11 +301,11 @@ fn run_daemon_command(args: &mut Vec<String>) -> io::Result<()> {
                     std::process::exit(EXIT_CODE_TIMEOUT_CONNECT_TO_RUNNING_DAEMON);
                 }
 
+                tries += 1;
```

Comment:
I noticed that the "tries" counter was moved and initialized differently. I do NOT claim that this is a problem. I remember that some similar loops displayed weird counting depending on whether it reached the end of the retries or not. I was just asking if you saw reasonable results displayed under all typical conditions during your testing.

Reply:
Previously the message "The Zowe daemon is in use, retrying (0 of 60)" was being shown.

I believe that moving the statement `tries += 1` fixes the range of X in "X of 60" to be from 1-60 rather than 0-59.